zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/client/TestRawZkClient.java

```diff
@@ -149,8 +149,7 @@ public void handleSessionEstablishmentError(Throwable error) {
    * Tests session expiry for the helix's IZkStateListener.
    */
   @Test
-  void testSessionExpiry()
-      throws Exception {
+  void testSessionExpiry() throws Exception {
```

Comment:
Nit, I saw this non-essential beautification all the time. Let us not add them to divert the attention.

Reply:
These are formatted by helix-style. I guess I formatted the whole file. Usually I don't do that. Well just a few lines I think.